### PR TITLE
[ASAP-401] - add analytics collaboration tab

### PR DESCRIPTION
**Pull request by AkosuaA**
ref: https://asaphub.atlassian.net/browse/ASAP-401

To display the Collaboration tab, enable the flag by entering in the console `document.cookie = 'ASAP_DISPLAY_ANALYTICS_COLLABORATION=true';`

### CODE REVIEW

File: apps/crn-frontend/src/analytics/collaboration/TeamCollaboration.tsx [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+      name: 'Team D',
+      isInactive: false,
+      OutputsCoProducedWithin: {
+        Article: 0,
+        Bioinformatics: 0,
+        Dataset: 0,
+        'Lab Resource': 0,
+        Protocol: 0,
+      },
+      OutputsCoProducedAcross: {
```
Comment (lctrt) [inline apps/crn-frontend/src/analytics/collaboration/TeamCollaboration.tsx:149]: mixing output types and collaboration details seems a bit confusing. 
Could this be
```
{OutputsCoproducedAcross:{
  Types: {
    Article: 0,
    ...
    },
    CollaborationDetails: []
   }

File: packages/model/src/analytics.ts [hunk 2]
```diff
@@ -85,3 +85,50 @@ export type ListTeamProductivityDataObject =
...
+  'Lab Resource': number;
+  Protocol: number;
+};
+type TeamCollaborationAcrossOutputData = {
+  Article: number;
+  Bioinformatics: number;
+  Dataset: number;
+  'Lab Resource': number;
+  Protocol: number;
+  'Collaboration Details': Array<
```
Comment (lctrt) [inline packages/model/src/analytics.ts:117]:
```suggestion
  collaborationDetails: Array<
```

File: packages/model/src/analytics.ts [hunk 3]
```diff
@@ -85,3 +85,50 @@ export type ListTeamProductivityDataObject =
...
+      isInactive: boolean;
+    } & TeamCollaborationWithinOutputData
+  >;
+};
+
+export type TeamCollaborationDataObject = {
+  id: string;
+  name: string;
+  isInactive: boolean;
+  OutputsCoProducedWithin: TeamCollaborationWithinOutputData;
```
Comment (lctrt) [inline packages/model/src/analytics.ts:130]:
```suggestion
  outputsCoproducedWithin: TeamCollaborationWithinOutputData;
```

**Review by lctrt — CHANGES_REQUESTED** [review]
Mostly good, just a few comments related to the data model

File: apps/crn-frontend/src/analytics/collaboration/TeamCollaboration.tsx (hunk 1 above)
Comment (AkosuaA) [reply apps/crn-frontend/src/analytics/collaboration/TeamCollaboration.tsx:149]: I could group them into 'byDocumentType' and 'byTeam'
i.e `{ OutputsCoProducedAcross: {
         byDocumentType: {
           Article: 0,
           ...
           },
         byTeam: []`} How does that sound?

**Review by lctrt — APPROVED** [review]

**Pull request merged**